cmd/azure-devops/smoke-tests/smoke-tests.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/kyma-project/test-infra/pkg/azuredevops/pipelines"
+)
+
+func main() {
+	// Fetching environment variables for Azure DevOps settings
+	organizationURL := os.Getenv("ORGANIZATION_URL")
+	personalAccessToken := os.Getenv("PERSONAL_ACCESS_TOKEN")
+	projectName := os.Getenv("PROJECT_NAME")
+	pipelineName := os.Getenv("PIPELINE_NAME")
+	pipelineIDStr := os.Getenv("PIPELINE_ID")
+	buildIDStr := os.Getenv("BUILD_ID")
+
+	// Converting variables from string to integer
+	pipelineID, err := strconv.Atoi(pipelineIDStr)
+	if err != nil {
+		log.Fatalf("Error parsing PIPELINE_ID: %v", err)
+	}
+	buildID, err := strconv.Atoi(buildIDStr)
+	if err != nil {
+		log.Fatalf("Error parsing BUILD_ID: %v", err)
+	}
+
+	// Setting up context for API calls
+	ctx := context.Background()
+
+	// Creating a build client using the Personal Access Token
+	buildClient, err := pipelines.NewBuildClient(organizationURL, personalAccessToken)
+	if err != nil {
+		log.Fatalf("Error creating build client: %v", err)
+	}
+
+	// Determining which tests to run based on the test-suite.yaml file
+	testsToRun := os.Getenv("FILE_PATH")
```

Comment:
Variable name could indicated expected file contains tests to run.
```suggestion
	testsToRun := os.Getenv("TESTS_TO_RUN_FILE_PATH")
```